Wave Animation/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en" >
 <head>
   <meta charset="UTF-8">
-  <title>CodePen - Coaster SVG</title>
+  <title>Wave Animation</title>
   <link rel="stylesheet" href="./style.css">
```

Comment:
@DragonUncaged  any clarifications ?

Reply:
I know it .
I store some of my work in codepen thats why the title was codepen.
As codepen doesn't need simple html body and other basic tag you can code directly

But When you export the file from codepen the codepen automatically adds all the basic tag required with the title codepen - your file name

Reply:
Some of my codes are from. Past which is not anywhere else.

Reply:
> @DragonUncaged any clarifications ?

.